homeassistant/components/light/zwave.py

```diff
@@ -20,6 +32,15 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     node = zwave.NETWORK.nodes[discovery_info[zwave.ATTR_NODE_ID]]
     value = node.values[discovery_info[zwave.ATTR_VALUE_ID]]
+    # Make sure that we have values for the key before converting to int
+    if (value.node.manufacturer_id.strip() and
```

Comment:
We should really create a helper method in the `zwave` component that you pass in your `value` and `WORKAROUNDS` constant and it will return `True` or `False`